### PR TITLE
topology page

### PR DESCRIPTION
**Pull request by mvoloshin0458**

### CODE REVIEW

File: src/utils.tsx [hunk 1]
```diff
@@ -161,8 +168,14 @@ const utils = {
...
-    let a = _getRowValuesByColumn4[0] || EMPTY_STRING
-    let b = _getRowValuesByColumn4[1] || EMPTY_STRING
+    let a = _getRowValuesByColumn4[0] ?? EMPTY_STRING
+    let b = _getRowValuesByColumn4[1] ?? EMPTY_STRING
+
+    if (typeof a === 'boolean' || typeof b === 'boolean') {
+      a = Number(a)
+      b = Number(b)
+    }
+
```
Comment (mvoloshin0458) [inline src/utils.tsx:178]: I'll need to discuss it

File: src/utils.tsx [hunk 2]
```diff
@@ -65,9 +66,15 @@ const utils = {
...
+        return objA[key]
+          .toString()
+          .toLowerCase()
+          .localeCompare(objB[key].toString().toLowerCase())
       }
-      return objA.toLowerCase().localeCompare(objB.toLowerCase())
+      return objA
+        .toString()
+        .toLowerCase()
+        .localeCompare(objB.toString().toLowerCase())
```
Comment (mvoloshin0458) [inline src/utils.tsx:77]: remove

File: src/utils.tsx (hunk 1 above)
Comment (mvoloshin0458) [reply src/utils.tsx:178]: remove

File: src/components/Table/table.scss [hunk 3]
```diff
@@ -330,4 +330,8 @@
       background-color: var(--icon-button-hover);
     }
   }
+
+  .loader-container {
+    height: 100%;
+  }
```
Comment (sofa-tata) [inline src/components/Table/table.scss:336]: can remove

File: src/components/Table/Table.tsx [hunk 4]
```diff
@@ -684,7 +731,9 @@ function Table({
           </table>
         </div>
       ) : (
-        <Loader />
+        <div className='loader-container'>
+          <Loader />
```
Comment (sofa-tata) [inline src/components/Table/Table.tsx:735]: can remove the div

**Review by sofa-tata — CHANGES_REQUESTED** [review]

**Comment by mvoloshin0458** [comment]
@sofa-tata tested in wekapp and gohome

**Review by sofa-tata — APPROVED** [review]

**Pull request merged**